client/src/main/java/io/split/client/SplitFactoryImpl.java

```diff
@@ -134,6 +137,7 @@ public class SplitFactoryImpl implements SplitFactory {
     private final ImpressionsSender _impressionsSender;
     private final URI _rootTarget;
     private final URI _eventsRootTarget;
+    private UniqueKeysTracker uniqueKeysTracker;
```

Comment:
I think this one can be final
```suggestion
    private final UniqueKeysTracker uniqueKeysTracker;
```

Reply:
I made it final.